package.json

```diff
@@ -24,7 +24,7 @@
     "test": "lerna run --ignore @cumulus/cumulus-integration-tests --concurrency 1 test",
     "test:ci": "./scripts/run_ci_unit_coverage.sh",
     "test:coverage": "lerna run --ignore @cumulus/cumulus-integration-tests --concurrency 1 test:coverage",
-    "coverage": "lerna run --ignore @cumulus/cumulus-integration-tests --concurrency 1 --since master coverage -- -- ",
```

Comment:
This apparently changed in the latest version of Lerna. This was the syntax to pass an argument to child processes (python).